--- a/bin/env
+++ b/bin/env
@@ -88,15 +88,15 @@
 fi
 
 # Active MQ installation dirs
-# ACTIVEMQ_HOME="<Installationdir>/"
+ACTIVEMQ_HOME="/usr/share/activemq/"
 # ACTIVEMQ_BASE="$ACTIVEMQ_HOME"
 # ACTIVEMQ_CONF="$ACTIVEMQ_BASE/conf"
-# ACTIVEMQ_DATA="$ACTIVEMQ_BASE/data"
-# ACTIVEMQ_TMP="$ACTIVEMQ_BASE/tmp"
+ACTIVEMQ_DATA="/var/lib/activemq/data"
+ACTIVEMQ_TMP="/var/lib/activemq/tmp"
 
 # Configure a user with non root privileges, if no user is specified do not change user
 # (the entire activemq installation should be owned by this user)
-ACTIVEMQ_USER=""
+ACTIVEMQ_USER="activemq"
 
 # location of the pidfile
 # ACTIVEMQ_PIDFILE="$ACTIVEMQ_DATA/activemq.pid"
