--- a/bin/env
+++ b/bin/env
@@ -25,9 +25,12 @@
 # Active MQ installation dirs
 # ACTIVEMQ_HOME="<Installationdir>/"
 # ACTIVEMQ_BASE="$ACTIVEMQ_HOME"
+ACTIVEMQ_HOME="/usr/share/activemq/"
 # ACTIVEMQ_CONF="$ACTIVEMQ_BASE/conf"
 # ACTIVEMQ_DATA="$ACTIVEMQ_BASE/data"
 # ACTIVEMQ_TMP="$ACTIVEMQ_BASE/tmp"
+ACTIVEMQ_DATA="/var/lib/activemq/data"
+ACTIVEMQ_TMP="/var/lib/activemq/tmp"
 
 # Set jvm memory configuration (minimal/maximum amount of memory)
 ACTIVEMQ_OPTS_MEMORY="-Xms64M -Xmx1G"
@@ -101,7 +104,7 @@
 
 # Configure a user with non root privileges, if no user is specified do not change user
 # (the entire activemq installation should be owned by this user)
-ACTIVEMQ_USER=""
+ACTIVEMQ_USER="activemq"
 
 # location of the pidfile
 # ACTIVEMQ_PIDFILE="$ACTIVEMQ_DATA/activemq.pid"
