--- a/bin/opensearch-env
+++ b/bin/opensearch-env
@@ -82,7 +82,7 @@
 export HOSTNAME=$HOSTNAME
 
 if [ -z "$OPENSEARCH_PATH_CONF" ]; then
-  if [ -z "$OPENSEARCH_PATH_CONF" ]; then OPENSEARCH_PATH_CONF="$OPENSEARCH_HOME"/config; fi
+  source /etc/default/opensearch
 fi
 
 if [ -z "$OPENSEARCH_PATH_CONF" ]; then
@@ -93,7 +93,7 @@
 # now make OPENSEARCH_PATH_CONF absolute
 OPENSEARCH_PATH_CONF=`cd "$OPENSEARCH_PATH_CONF"; pwd`
 
-OPENSEARCH_DISTRIBUTION_TYPE=tar
+OPENSEARCH_DISTRIBUTION_TYPE=deb
 OPENSEARCH_BUNDLED_JDK=true
 
 if [[ "$OPENSEARCH_BUNDLED_JDK" == "false" ]]; then
