--- a/build/opencast-dist-base/docs/scripts/service/opencast.service
+++ b/build/opencast-dist-base/docs/scripts/service/opencast.service
@@ -8,8 +8,8 @@
 Documentation=https://docs.opencast.org
 
 [Service]
-ExecStart=/opt/opencast/bin/start-opencast server
-ExecStop=/opt/opencast/bin/stop-opencast
+ExecStart=/usr/share/opencast/bin/start-opencast server
+ExecStop=/usr/share/opencast/bin/stop-opencast
 Restart=always
 User=opencast
 
