--- build/opencast-dist-base/docs/scripts/service/etc-init.d-opencast
+++ build/opencast-dist-base/docs/scripts/service/etc-init.d-opencast
@@ -14,8 +14,8 @@
 # Provides: opencast
 # Required-Start: $local_fs $remote_fs $syslog $network
 # Required-Stop:
-# Default-Start:
-# Default-Stop:
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
 # Short-Description: run opencast
 # Description: Opencast is a free, open-source platform to support the
 #              management of educational audio and video content. Institutions
@@ -25,14 +25,14 @@
 #              videos.
 ### END INIT INFO
 
-opencast="/opt/opencast"
+opencast="/usr/share/opencast"
 prog="opencast"
 user="opencast"
 lockfile=/var/lock/subsys/${prog}
 [ -d "/var/lock/subsys" ] || lockfile="/var/lock/LCK.${prog}"
 
 # Load configuration files
-[ -e /etc/sysconfig/$prog ] && . /etc/sysconfig/$prog
+[ -e /etc/default/$prog ] && . /etc/default/$prog
 
 success() {
    printf "\r%-58s [\033[32m  OK  \033[0m]\n" "$1"
--- build/opencast-dist-base/docs/scripts/service/opencast.service
+++ build/opencast-dist-base/docs/scripts/service/opencast.service
@@ -5,8 +5,8 @@
 After=remote-fs.target
 
 [Service]
-ExecStart=/opt/opencast/bin/start-opencast server
-ExecStop=/opt/opencast/bin/stop-opencast
+ExecStart=/usr/share/opencast/bin/start-opencast server
+ExecStop=/usr/share/opencast/bin/stop-opencast
 Restart=always
 User=opencast
 
