--- build/opencast-dist-base/etc/org.ops4j.pax.logging.cfg
+++ build/opencast-dist-base/etc/org.ops4j.pax.logging.cfg
@@ -25,6 +25,6 @@
 log4j.appender.out=org.apache.log4j.FileAppender
 log4j.appender.out.layout=org.apache.log4j.PatternLayout
 log4j.appender.out.layout.ConversionPattern=%d{ISO8601} | %-5.5p | (%C{1}:%L) - %m%n
-log4j.appender.out.file=${karaf.data}/log/opencast.log
+log4j.appender.out.file=/var/log/opencast/opencast.log
 log4j.appender.out.append=true
 log4j.appender.out.encoding=UTF-8
